packages/perennial/contracts/multiinvoker/MultiInvoker.sol

```diff
@@ -214,7 +269,7 @@ contract MultiInvoker is IMultiInvoker, UInitializable {
      * @param product Product to withdraw funds for
      * @param amount Amount of DSU to withdraw from the collateral account
      */
-    function withdrawAndUnwrap(address receiver, IProduct product, UFixed18 amount) private {
+    function _withdrawAndUnwrap(address receiver, IProduct product, UFixed18 amount) internal {
```

Comment:
This needs to call `_handleUnwrap` internally now, since we changed `_unwrap` to `_handleUnwrap`

Same for `_handleWrap`

Reply:
yeah looks like this is coming up in the integration tests